### ISSUE TITLE
Tags not being added to workflow when imported through command line

### ISSUE THREAD

**Issue by marlom-rvvup** [n8n-io/n8n#5477]
**Describe the bug**
When importing a workflow the `tags` are being created when new, but they are not being set to the workflow.

**To Reproduce**

Create a simple workflow, export it to json, and add the following `tags`: 
```
    "tags": [
      {
        "createdAt": "2023-02-14T15:33:32.302Z",
        "updatedAt": "2023-02-14T15:33:32.302Z",
        "id": "123123123",
        "name": "my-new-tag"
      }
    ]
```

AFter import it using the CLI: `n8n import:workflow --input=workflow.json

**Expected behavior**

When importing this workflow, I expect:
- The tag to be created if it doesn't exist.
- The tag to be linked to the workflow.

**Environment (please complete the following information):**

- OS:MacOS 13.2
- n8n Version 0.215.1
- Node.js Version v18.12.1
- Database system MySQL
- Operation mode `own` (default)

**Comment by Joffcom** [comment]
Hey @marlom-rvvup,

Thanks for reporting this one, I have managed to reproduce it and created an internal ticket which we will be tracking as `PAY-221`

**Comment by Lucaber** [comment]
The issue is probably [this](https://github.com/n8n-io/n8n/pull/5231/files#diff-ff45b1d8db8d3dc67d19aa34e4d066e4156ddcd311c0b8d75a3d082009065197R212) change from `save` to `upsert` of the workflow entity.

The typeorm `upsert` docs state `Unlike save method executes a primitive operation without cascades, relations and other operations included.`

This prevents the tag relations from being inserted into the database.

**Comment by Joffcom** [comment]
@Lucaber that could be it, It looks like the issue is currently in backlog with no engineer assigned so I wouldn't worry about tagging Netroy I suspect with the notifications he gets it will be lost 😉

**Comment by netroy** [comment]
The code changes from #5231 aren't relevant anymore, since the 0.234.x release, because of how n8n does IDs now. 
@Lucaber @marlom-rvvup Do either of you still see this issue in the latest release ?

**Comment by Lucaber** [comment]
I can replicate this issue in the current master branch. This means we should now (with the new id schema) be able to revert https://github.com/n8n-io/n8n/pull/5231 (at least the workflow upsert part) to fix this issue

**Comment by janober** [comment]
Fix got released with [`n8n@1.21.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.21.0)
